control/available_plugin.go

```diff
@@ -285,22 +285,15 @@ func (a *availablePlugin) healthCheckFailed() {
 		"aplugin": a,
 	}).Warning("heartbeat missed")
 	a.failedHealthChecks++
-	if a.failedHealthChecks >= DefaultHealthCheckFailureLimit {
-		log.WithFields(log.Fields{
-			"_module": "control-aplugin",
-			"block":   "check-health",
-			"aplugin": a,
-		}).Warning("heartbeat failed")
-		pde := &control_event.DeadAvailablePluginEvent{
-			Name:    a.name,
-			Version: a.version,
-			Type:    int(a.pluginType),
-			Key:     a.key,
-			Id:      a.ID(),
-			String:  a.String(),
-		}
-		defer a.emitter.Emit(pde)
+	pde := &control_event.DeadAvailablePluginEvent{
+		Name:    a.name,
+		Version: a.version,
+		Type:    int(a.pluginType),
+		Key:     a.key,
+		Id:      a.ID(),
+		String:  a.String(),
 	}
+	defer a.emitter.Emit(pde)
```

Comment:
Do we want to send a DeadAvailablePluginEvent even though we may or may not have passed the health check failure limit?